package.json

```diff
@@ -27,28 +25,20 @@
          "properties": {
             "aks.draft.releaseTag": {
                "type": "string",
-               "default": "v0.0.22",
```

Comment:
💡 Package version update could be part of `release` PR not the code PR.

Reply:
Thank you for review! 

Aligned on the package version update being part of the release PR. There's not package version update happening in this PR currently. 

The quoted text is the pinned Draft version tag which needs to be bumped in this PR since we are relying on features from new Draft versions.